test/integrationTests/hoverProvider.integration.test.ts

```diff
@@ -0,0 +1,63 @@
+
+import * as fs from 'async-file';
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+import poll from './poll';
+import { should, expect } from 'chai';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+
+const chai = require('chai'); 
+chai.use(require('chai-arrays')); 
+chai.use(require('chai-fs')); 
+
+suite(`Test Hover Behavior ${testAssetWorkspace.description}`, function() {
+    suiteSetup(async function() { 
+        should();
+
+        let csharpExtension = vscode.extensions.getExtension("ms-vscode.csharp"); 
+        if (!csharpExtension.isActive) { 
+            await csharpExtension.activate();
+        }
+
+        testAssetWorkspace.deleteBuildArtifacts();
+        
+        await fs.rimraf(testAssetWorkspace.vsCodeDirectoryPath);
+        
+        await csharpExtension.exports.initializationFinished;
+                
+        await vscode.commands.executeCommand("dotnet.generateAssets");
+        
+        await poll(async () => await fs.exists(testAssetWorkspace.launchJsonPath), 10000, 100);
+
+    });
+
+
+    test("Hover returns the correct XML", async () => {                
+        
+        let program : string = "using System;\
```

Comment:
Can we use a [multi-line string](https://basarat.gitbooks.io/typescript/content/docs/template-strings.html) instead? it should be much easier to maintain since we won't be  in the business of moving around `\n`'s and `\`'s.

Reply:
+1